e2e_samples/parking_sensors/src/ddo_transform/Dockerfile

```
@@ -1,9 +1,14 @@
-FROM python:3.7.3
+FROM python:3.11
```

This was the change that we used to address different architecture on the main container - can't we use the same so it works on MAC as well?
```suggestion
FROM --platform=linux/amd64 python:3
```

---

updated.